.github/workflows/lint.yml

```diff
@@ -24,7 +24,14 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
-      run:  pip install tox
+      run:  pip install poetry tox
+    - name: Check Poetry lock file
+      run: |
+        poetry check
+        echo "Info: 'poetry check' passed. Now trying to update the lock file."
+        poetry lock
+        git diff --exit-code poetry.lock || (echo "Error: poetry.lock file has changed. Please update it and commit the changes." && exit 1)
+        echo "Info: poetry.lock file is up-to-date with pyproject.toml."
```

Comment:
@muxator is it OK for you?

Reply:
Talked offline, we'll replace it with a `poetry check --lock`.